src/main/java/graphql/schema/DataFetchingEnvironment.java

```diff
@@ -170,4 +171,6 @@
      * @see org.dataloader.DataLoaderRegistry#getDataLoader(String)
      */
     <K, V> DataLoader<K, V> getDataLoader(String dataLoaderName);
+
+    EncounterDirectives getEncounterDirectives();
```

Comment:
Eventually this needs JavaDoc and also the name cchange mentioned above to `getEncounteredDirectives` ;)